### ISSUE TITLE
[non-issue] Notifications, media player and MQTT

### ISSUE THREAD

**Issue by kvikindi** [LAB02-Research/HASS.Agent-Integration#31]
Just leaving this info for anyone that might have the same problems I did.

I struggled to get a new computer set up with hass.agent and the integration.
Namely, sensors etc. would work, but MQTT/notifications and media player weren't working.
After an utterly silly amount of time trying to debug this and that... Turns out that all I needed to do was rename my computer in hass.agent's setting from lower case to all caps...

**TL;DR:** If you're struggling with notifications, media player and MQTT, try changing your hostname from all lower case to ALL CAPS.
